src/helpers/index.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
   defaultProps: require('./defaultProps'),
   defaultTo: require('./defaultTo'),
   dissoc: require('./dissoc'),
-  fanout: require('./fanout'),
+  fanout: require('../Pair/fanout'),
```

Comment:
I think you were pointing at an old ref in your comment.
We no longer export cross folder in those index files.
This will need to be removed from this index file and referenced solely in the main crocks index file.